add_partition.sh

```diff
@@ -0,0 +1,55 @@
+#/bin/bash
```

Comment:
Nice! A bash script! 🙌 I suggest using shellcheck and sh on this.
https://github.com/mvdan/sh
https://www.shellcheck.net/#

If possible, please also add a comment block at the top explaining what this is for. I assume this is for CloudLab setup stuff?

Reply:
right, this is for extending the partition for cloudlab machine because there is only 16G for / path in cloudlab machine, very easy to exhaust the disk space if you create many docker containers, we usually extend the / space when we use cloudlab machine.